um_utils/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: um_utils
+    version: 1.2
+
+source:
+    fn: um_utils.tar.gz
+    url: https://github.com/SciTools/um_utils/releases/download/vn1.2/um_utils.tar.gz
+
+build:
+    number: 0
+    skip: True # [py3k or win]
```

Comment:
@stevewardle Minor tweak ... for this `um_utils` recipe and the above `mule` recipe you could opt to remove the associated `build.sh` file (which is pretty light weight) and include the build script in-line within the `build` section e.g.
```shell
build:
    number: 0
    skip: True  # [py3k or win]
    script: python setup.py install --single-version-externally-managed --record record.txt
```